### DISCUSSION TITLE
Taskbar not immediately available after login

### DISCUSSION THREAD

**Discussion by udobehm** [valinet/ExplorerPatcher#853] [General]
Hi, when I log into Windows I need to wait for about 10 - 15 seconds until my Windows 10 taskbar has loaded under Win 11.

Can you somehow make the taskbar be accessible and useable immediately when it appears first??

**Comment by valinet** [comment]
I don’t understand exactly. For me, it works fine. In Properties - Advanced there is a setting called “Supplimentary delay at logon”. You can use that to increase the time until the login screen is dismissed.

By default, it waits until the taskbar is created. If it takes too long for it to be actually useful, that could be due to a couple of things, like too slow backing storage, maybe slow extensions that have to load etc. On a regular config, it works just fine.

**Comment by valinet** [comment]
Also, please follow the advice here  when posting here: https://github.com/valinet/ExplorerPatcher/wiki/Reporting-problems

**Reply by udobehm** [reply]
Hmm, I don't know. The value set under your suggested option is set to none here. It takes about 18 seconds until all of the symbols get shown and until I can click anything on the taskbar. Initially only these symbols become loeaded, but I cannot click on any of them until all the symbols are loaded correctly:
![Screenshot 2022-02-13 161020](https://user-images.githubusercontent.com/50866188/153759808-02508542-940a-4bae-aaab-ae696681d3f3.jpg)

I was using StartAllBack before. With that app I didn't have that problem. Symbols and the taskbar were shown and accessible immediately. Would like to have access as fast as with the StartAllBack app with your Explorer Patcher app.

**Reply by valinet** [reply]
It should work as fast as that. It does here. I don’t have any delay at logon. On my system it works; without any additional insight, I cannot guess what might be happening over there. I don’t know…

**Comment by udobehm** [comment]
Hm, too bad. You got no clue what could cause the delay and how to avoid it?
As said - it was not the case when I still was using the StartAllBack app...
________________________________

**Reply by valinet** [reply]
I understand about SAB, but like, that doesn’t help, I still don’t know. Maybe he does something else at startup, I simply don’t know. And right now I can’t think of anything that might be happening. Try restoring default settings, idk. You’ll see, if you install EP on a clean VM, it will work just fine. Try to investigate it a bit, I am not a wizard, I cannot guess things.

**Comment by udobehm** [comment]
I've just followed your suggestion and set all the settings to default.

Then I've figured out that there is the same delay when I click on the Restart File Explorer entry in your configuration tool. The taskbar then loads faster than during the Windows startup. But also, first the outer icons (as on my screenshot) get loaded. The program icons take a while until they become loaded as well as the symbols in the tray. I cannot use the taskbar immediately...

**Reply by valinet** [reply]
Yeah, idk. "Restart File Explorer" reloads `explorer` the same way it is reloaded at logon. Try in a VM and see how it works there. Then you'll know whether it is a general problem or just one local to your configuration. And from there, idk, try to revert some settings and maybe isolate what could be the culprit. As I said, I cannot guess, and don't really do anything that might hang the load process. For me, it loads instantly, I mean, fast, without any noticeable delay.

**Comment by udobehm** [comment]
You think it could have to do that Windows prevents startup apps from loading for the first 10 seconds after the OS got loaded?
